### DISCUSSION TITLE
How far it was to Manchester?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#910] [Galaxy Trivia] [answered]
Last night, I was looking at a road sign and could see that Leeds was 13 miles away, Stoke was 13 miles away and Leek was 10 miles away.

Unfortunately, it was too dark to see...

**Comment by peterbaccaro** [comment, marked as answer]
Leeds = 13 miles
Stoke = 13 miles
Leek = 10 miles

This tells us that vowels are worth 2 miles and consonants are worth 3 miles.

Manchester has 3 vowels and 7 consonants, (3 x 2) + (7 x 3) = 27.

Manchester = **27 miles**

**Comment by raeef1001** [comment]
Leeds = 13 miles
Stoke = 13 miles
Leek = 10 miles

This tells us that vowels are worth 2 miles and consonants are worth 3 miles.

Manchester has 3 vowels and 7 consonants, (3 x 2) + (7 x 3) = 27.

Manchester = 27 miles
